services/udp/config.go

```diff
@@ -0,0 +1,13 @@
+package udp
+
+import "github.com/influxdb/influxdb/toml"
+
+type Config struct {
+	Enabled    bool   `toml:"enabled"`
+	BindAdress string `toml:"bind-adress"`
+	Port       int    `toml:"port"`
```

Comment:
No longer using `Port` in our configs.  BindAddress should now be in the format of `host:port`